src/main/java/catan/domain/transfer/output/game/GameUserDetails.java

```diff
@@ -8,15 +8,19 @@
     private int colorId;
     private boolean ready;
     private int moveOrder;
+    private ResourcesDetails resources;
 
     public GameUserDetails() {
     }
 
-    public GameUserDetails(GameUserBean userBean) {
+    public GameUserDetails(GameUserBean userBean, int userId) {
```

Comment:
the same here, please rename parameter to understand in code, what does it mean and why we need to check it)